docs/reference/troubleshooting/common-issues/hotspotting.asciidoc

```diff
@@ -0,0 +1,266 @@
+----
+GET _cat/allocation?v&s=node&h=node,shards,disk.percent,disk.indices,disk.used
+----
+
+Which could return:
+
+[source,console-result]
+----
+node   shards disk.percent disk.indices disk.used
+node_1    446           19      154.8gb   173.1gb
+node_2     31           52       44.6gb   372.7gb
+node_3    445           43      271.5gb   289.4gb
+----
+// TEST[skip:illustrative response only]
+
+Here we see two significantly unique situations. `node_1` has 
+recently restarted, so it has a much lower number of shards than all other nodes. 
+This also relates to `disk.indices` being much smaller than `disk.used` 
+while shards are recovering as seen via <<cat-recovery,cat recovery>>. The 
+second situation is that `node_3` has a higher `disk.percent` than `node_1`, 
+even though they hold roughly the same number of shards. This 
+occurs when either shards are not evenly sized (refer to 
+<<shard-size-recommendation>>) or when there are a lot of empty indices.
+
+Cluster rebalancing based on desired balance does much of the heavy lifting 
+of keeping nodes from hot spotting. It can be limited by either nodes hitting 
+<<disk-based-shard-allocation,watermarks>> (refer to <<fix-watermark-errors,fixing disk watermark errors>>) or by a 
+write-heavy index's total shards being much lower than the written-to nodes. 
+
+You can confirm hot spotted nodes via <<cluster-nodes-stats,the nodes stats API>>, 
+potentially polling twice over time to only checking for the stats differences 
+between them rather than polling once giving you stats for the node's 
+full <<cluster-nodes-usage,node uptime>>. For example, to check all nodes 
+indexing stats:
+
+[source,console]
+----
+GET _nodes/stats?human&filter_path=nodes.*.name,nodes.*.indices.indexing
+----
+
+[discrete]
+[[causes-shards-index]]
+====== Index level
+
+Hot spotted nodes frequently surface via <<cat-thread-pool,cat thread pool>>'s 
+`write` and `search` queue backups. For example:
+
+[source,console]
+----
+GET _cat/thread_pool/write,search?v=true&s=n,nn&h=n,nn,q,a,r,c
+----
+
+Which could return:
+
+[source,console-result]
+----
+n      nn       q a r    c
+search node_1   3 1 0 1287
+search node_2   0 2 0 1159
+search node_3   0 1 0 1302
+write  node_1 100 3 0 4259
+write  node_2   0 4 0  980
+write  node_3   1 5 0 8714
+----
+// TEST[skip:illustrative response only]
+
+Here you can see two significantly unique situations. Firstly, `node_1` has a severely 
+backed up write queue compared to other nodes. Secondly, `node_3` shows historically 
+completed writes that are double any other node. These are both probably due 
+to either poorly distributed write-heavy indices, or to multiple write-heavy 
+indices allocated to the same node. Since primary and replica writes 
+are majorly the same amount of cluster work, we usually recommend setting 
+<<total-shards-per-node,`index.routing.allocation.total_shards_per_node`>> 
+to force index spreading after lining up index shard counts to total nodes. 
+
```

Comment:
@abdonpijpelink could you help me add two more common recoveries I missed (from support feedback); something to the effect:

```
We normally recommend heavy-write indices have sufficient primary 
`number_of_shards` and replica `number_of_replicas` to evenly spread 
across ingesting nodes. Alternatively, you can <<cluster-reroute,Reroute>> 
shards to more quiet nodes to alleviate the nodes with write hot spotting.
```